modules/installation-configuration-parameters.adoc

```diff
@@ -407,6 +407,15 @@ endif::ibm-power-vs[]
 Set the `networking.machineNetwork` to match the CIDR that the preferred NIC resides in.
 ====
 
+|networking:
+  ovnKubernetesConfig:
+    ipv4:
+      internalJoinSubnet:
+|
+Configures the IPv4 join subnet that is used internally by `ovn-kubernetes`. This subnet must not overlap with any other subnet that {product-title} is using, including the node network. The size of the subnet must be larger than the number of nodes. You cannot change the value after installation.
+
+|An IP network block in CIDR notation. For example, `10.0.0.0/16`. The default value is `100.64.0.0/16`.
```

Comment:
nit: I read `For example, `10.0.0.0/16`.` as boiler plate language we use with every cidr. So I'm fine leaving it in for consistency if that is the case. On the other hand, 10.0.0.0/16 would never be an appropriate value (10.0.0.0/16 would be for the machine network), so I think it is just as good, or a slight improvement to say:

```suggestion
|An IP network block in CIDR notation.  The default value is `100.64.0.0/16`.
```